### ISSUE TITLE
Change the menu link to the ARexx documentation

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#434]
**Originally by _AmigaPhil@scarlet.be_ on 2013-09-24 22:42:10 +0200**

___
I have finished to convert the ARexx documentation from AmigaGuide to online wiki documents.

The main ARexx page is no more rf01 like the node in the guide, but ARexx%20API
The http link is to be changed in the menu > Help > ARexx interface

**Comment by tboeckel** [comment]
**Originally on 2013-09-27 19:46:59 +0200**

___
In (7e0b173):
 * YAM_MA.c: changed the ARexx menu item help node to the new Wiki link. This closes #434.

**Comment by jens-maus** [comment]
**Originally on 2013-12-23 16:41:09 +0100**

___
**Release Note:**
the online ARexx documentation (http://doc.yam.ch/) can now be directly accessed via the corresponding main menu item.

**Comment by jens-maus** [comment]
**Originally by _AmigaPhil@scarlet.be_ on 2014-04-04 14:14:20 +0200**

___
The main page for ARexx documentation has changed again.
The new name is ARexxAPI (without the space separator "%20").

**Comment by jens-maus** [comment]
**Originally by _AmigaPhil@scarlet.be_ on 2014-04-04 14:17:13 +0200**

___
Sorry, the new name is more exactly: "ARexxAPI", with a capital "R" (I've edited the previous message).

**Comment by jens-maus** [comment]
**Originally on 2014-04-04 15:13:27 +0200**

___
Replying to AmigaPhil:
> The main page for ARexx documentation has changed again.
> The new name is ARexxAPI (without the space separator "%20").

Sorry, but have you actually checked that it is not working with the latest nightly build? Indeed I already fixed that in the recent nightly builds when I changed the name of the page. Thus, with one of the latest nightly builds the main menu link should properly work. Thus, there is no need for any fix as the upcoming 2.9p1 version will fix it immediately.
